src/dbFunctions.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+function connectToDb(string $dbname): PDO {
+    $db = new PDO("mysql:host=db; dbname=$dbname", "root", "password");
+    $db -> setAttribute(PDO::ATTR_DEFAULT_FETCH_MODE, PDO::FETCH_ASSOC);
+    
+    return $db;  
+}
+
+function getItemsFromDb(PDO $db): array {
+    $query = $db->prepare("SELECT * FROM yarn");
```

Comment:
according to best practice the table should be called `yarns`

Reply:
I would also say you shouldn't do use the * here - it's ok when quickly testing something, but it's better to specifically list the fields you want to select, even if it's all of them.

The reasoning is that the table might expand in the future to contain more fields that you may not want to retrieve in this particular query.

Reply:
done!